packages/main/src/themes/base/sizes-parameters.css

```diff
@@ -287,6 +287,7 @@
 	--_ui5_load_more_text_height: 2.625rem;
 	--_ui5_load_more_text_font_size: var(--sapFontSize);
 	--_ui5_load_more_desc_padding: 0 2rem 0.875rem 2rem;
+	--ui5_table_header_row_height: 2rem;
```

Comment:
In this file, the default values for sap_fiori_3 theme are stored. So I think here it should be 2.75rem. And for the theme where you need it to be 2rem, you should add it in the corresponding file

Reply:
That is the corresponding value for compact density which is the same for all supported themes.